spec/components/unranked_suppliers/dropdown_component_spec.rb

```diff
@@ -5,7 +5,16 @@
 RSpec.describe UnrankedSuppliers::DropdownComponent, type: :component do
   subject { page }
 
-  let(:suppliers) { build_list(:supplier, 10) }
+  # let(:suppliers) { build_list(:supplier, 10) }
+  let(:suppliers) do
+    build_list(:supplier, 10) do |record, i|
+      # generate unique suppliers by adding a suffix letter
+      # then shuffle out of alphabetical order
+      record.data.name = "An Energy Supplier Inc #{(i + 65).chr}"
```

Comment:
This fine and I can infer what's happening.  `+65` and `+97` feel a bit like magic numbers/strings to me.

I wonder if
```
record.data.name = "An Energy Supplier Inc #{('a'..'z').to_a[i]}"
```

is more readable?  I do love [ruby ranges](https://www.rubyguides.com/2016/06/ruby-ranges-how-do-they-work/) though so maybe I'm being biased!